packages/@aws-cdk/aws-codepipeline-actions/package.json

```diff
@@ -85,6 +85,7 @@
     "@aws-cdk/aws-events-targets": "0.0.0",
     "@aws-cdk/aws-iam": "0.0.0",
     "@aws-cdk/aws-lambda": "0.0.0",
+    "@aws-cdk/aws-servicecatalog": "0.0.0",
```

Comment:
1. These should be in alphabetical order, so can you move this exactly one line down?
2. Should be added to `peerDependencies` below as well.